backend/tests/test_integration_org_unit.py

```diff
@@ -2832,3 +2832,68 @@ def test_tree(self):
         for path, expected in util.get_fixture('test_trees.json').items():
             with self.subTest(path):
                 self.assertRequestResponse(path, expected)
+
+    def _create_conf_data(self):
```

Comment:
I think we should move configuration testing to a separate file, such as `test_integration_configuration.py`, as this one is already quite long indeed.